playbooks/support/scheduling.md

```diff
@@ -107,5 +79,49 @@ To override a shift:
-[artsy-cli]: https://github.com/artsy/artsy-cli
+## **Adding Engineers**
+
+Generally, new engineers should be added to the end of the schedule rotation associated with their time zone. To do this:
+
+1. If the engineer doesn't have an Opsgenie account already, [create an one for them](/Users/mattjones/code/README/playbooks/support/scheduling.md) in the Settings -> Users page, then [add them](https://support.atlassian.com/opsgenie/docs/create-a-team-and-add-members/)to the Engineering team in the Engingeering members page (Teams -> Engineering -> Members).
+
+2. Navigate to the [On-Call Schedule](https://artsy.app.opsgenie.com/teams/dashboard/ee381004-a72e-42ef-a733-b350d6693c6c/main) and edit the engineer's respective rotation.
+
+   <img width=500 src="images/opsgenie-addengs-1.png" />
+   <img width=500 src="images/opsgenie-addengs-2.png" />
+
+3. Add the engineer to the end of the participants list. Their position in the rotation will match their position in the participants list on the rotation edit page, for example:
+
+Considerations when adding a new engineer:
+
+- A new hire should be given 60 days after their start date to get the necessary context before having an on-call shift. If a new engineer's start date is within 60 days of the current rotation ending, then you should delay adding them until the new rotation has started. You can identify when a new rotation ends by checking the end date of the last participant's next rotation:
+
+   <img width=500 src="images/opsgenie-addengs-3.png" />
+
+   - We recommend setting a [Slack reminder](https://slack.com/blog/productivity/never-forget-the-little-things-with-remind) or calendar event for yourself. In Slack you can:
+
+      > /remind me to add @new-engineer to on-call rotations in 60 days
+
+- You should aim to avoid pairings with two new engineers and can place the new engineer earlier in the rotation to avoid this.
+- **NOTE: If you add engineers anywhere other than the end of the rotation, the schedule of the engineers who have not completed their shifts will be modified, as the order of the participants matters. You should therefore update the affected engineers to avoid scheduling conflicts. You can message them something like:**
+
+   > Hi @engingeer-whose-schedule-shifted! We've added a new engineer to the on-call schedule which has modified the dates of your on-call shift. Please take some time to review the schedule and trade shifts as needed.
+
+   - In certain scenarios, to minimize shifting multiple engineers schedules, it may be preferable to reach out to one engineer directly and ask them to switch shifts. Something like:
+
+      > Hi @engineer! We've added a @new-engineer to the schedule and would like to support them by not pairing them with another new engineer for their first on-call shift. Would you be able to switch shifts with them? The new dates of your on-call shift would be X to Y.
+
```

Comment:
I'd like this whole section to be more fool-proof somehow.
* Can we advise adding new members immediately to (a) the first `no one` slot that's >60 days out, (b) the end of the rotation if >60 days out, or (c) in the case the rotation is ending sooner, repeating (a) and (b) once the new rotation starts?
* If we do that, can we altogether avoid shifting near-term rotation schedules, and the necessary communication?

Reply:
See above comment. Happy to switch to this if we feel this is a better approach than removing the no-one placeholders.

But let me know if my point is unclear.

Reply:
My thinking is:
* Even with this approach, there _will_ be `no one` placeholders in the schedule (between engineers departing and their next [overridden] shift). If they're there (until the manager returns after the shift to remove them), why not take advantage of them to randomize the new additions?
* Otherwise, won't we necessarily lump all of the newest team members together at the end of the rotation necessitating various judgment calls, communication, and adjustments?

Reply:
As mentioned above, I agree this makes more sense, give the shifting that happened when the no-one placeholder is removed. I'll incorporate the suggestions you make here.

Reply:
@joeyAghion I Have incorporated the changes but left the 'NOTE' bullet. I want to explicitly call out there is a cascading effect if the event an engineering manager does need to put in a new location for some reason.